source/Planet.cpp

```diff
@@ -388,6 +392,22 @@ const Sale<Ship> &Planet::Shipyard() const
 
 
 
+// Get the shipyard description
+string Planet::ShipyardDescription() const
+{
+	if(!shipyardDescription.empty())
+		return shipyardDescription;
+
+	string shipyardDescriptionFromSales = "";
```

Comment:
I don't think you need to explicitely say its an empty string, just declaring it will make it be default I think